ament_cmake_google_benchmark/ament_cmake_google_benchmark/__init__.py

```diff
@@ -40,9 +40,16 @@ def main(argv=sys.argv[1:]):
 
     res = subprocess.run(args.command)
 
-    with open(args.result_file_in, 'r') as in_file:
-        with open(args.result_file_out, 'w') as out_file:
-            # TODO(cottsay): Convert the results file
-            pass
+    try:
+        with open(args.result_file_in, 'r') as in_file:
+            with open(args.result_file_out, 'w') as out_file:
+                # TODO(cottsay): Convert the results file
+                pass
+    except FileNotFoundError:
+        print(
+            'ERROR: No performance test results were found at: %s' % args.result_file_in,
```

Comment:
@cottsay meta: does it make sense to log such an error if the subprocess execution failed completely?

Reply:
As is, the message is written if the results file wasn't generated, which doesn't necessarily mean that the test failed or that the process returned non-zero. A configuration problem could have resulted in the result file being written to the wrong place, for example.

Reply:
I changed the code to only print the message if the subprocess succeeded, so we're now depending on that subprocess to write error messages properly when it returns non-zero. But we shouldn't get an excess message when the process fails properly now.